api/pom.xml

```diff
@@ -171,7 +171,7 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>2.8.1</version>
```

Comment:
Does it make sense upgrading to version 3.1.0? You will be able to build on Java 11 in this case, but possibly will need to fix some javadocs.

Reply:
Perhaps in the future.
I'd like to keep the changes to an absolute minimum in the short term.

Reply:
It's not a blocking issue. I'm fine with leaving it as it is for now.